### DISCUSSION TITLE
PLS and DA

### DISCUSSION THREAD

**Discussion by ruben-ad-araujo** [biolab/orange3#5217] [Q&A]
Hello everyone! 

I was wondering if anyone knows how to properly use the PLS widget? Since it has no documentation I cannot for the life of me make it to work. I have used for Spectroscopy a specialized software (The Unscrambler X), but I really like Orange, as it is easier to show people and can produce faster imagery and easier on the eyes. Alas, as it has no documentation, I cannot make it work no matter how much I try or browse the net.

Any thoughts? Thank you!

**Comment by ajdapretnar** [comment]
@markotoplak Time for some doc updates?

**Reply by ruben-ad-araujo** [reply]
Haha, that would be nice. And thank you ajdapretnar. It is nice to see the discussion is alive :)

**Comment by markotoplak** [comment]
The PLS currently in Orange does not have any visualization and inspection capabilities of the one in the Unscrambler, which is a fantastic tool for PCA and PLS.

The Orange implementation can only be used as a predictive method, which means it can be used in the same way as a Logistic regression widget. It currently has no ways to inspect the PLS scores or components, so it is very unlikely that it will be usable for you at this stage. 

We are planning to update it with more capabilities and will then add the documentation.

**Reply by ruben-ad-araujo** [reply]
Thank you so much for the reply @markotoplak! I am also pleased to have found someone who knows about Unscrambler! :) I will look forward to future updates and to have myself the ability to help you guys in the future! Again, thank you!

**Reply by borondics** [reply]
If you would like, please feel free to go ahead and add code to the PLS widget! :)

**Comment by borondics** [comment]
@Benruren, thanks for your interest in our work! I am glad that you like the functionalities. As @markotoplak says, the PLS (PLS-DA eventually) functionality is in development still but we hope it will come to a fully capable replacement for the Unscrambler features. If you want to stay in touch, we have a newsletter that comes when we have a new release for the package. 

You can sign up at [https://quasar.codes/contact](https://quasar.codes/contact).

**Reply by ruben-ad-araujo** [reply]
Thank you, @borondics I will do just that, and drag my lab mates into it :) Thank you!

**Reply by irgolic** [reply]
If you got any questions, feel free to ask here or on [Discord](https://discord.com/invite/FWrfeXV). We're happy for any feedback on our widget development documentation.

**Reply by ruben-ad-araujo** [reply]
@irgolic thank you! I will do that! :)
